packages/ember-data/lib/main.js

```diff
@@ -4,14 +4,17 @@
   @main ember-data
 */
 
-if (Ember.VERSION.match(/^1\.[0-7]\./)) {
-  throw new Ember.Error("Ember Data requires at least Ember 1.8.0, but you have " +
+if (Ember.VERSION.match(/^1\.(1[0-2]|[0-9])\./)) {
+  throw new Ember.Error("Ember Data requires at least Ember 1.13.0, but you have " +
                         Ember.VERSION +
-                        ". Please upgrade your version of Ember, then upgrade Ember Data");
+                        ". Please upgrade your version of Ember, then upgrade Ember Data.");
 }
 
-if (Ember.VERSION.match(/^1\.12\.0/)) {
-  throw new Ember.Error("Ember Data does not work with Ember 1.12.0. Please upgrade to Ember 1.12.1 or higher.");
+if (Ember.VERSION.match(/^1\.13\./)) {
+  Ember.deprecate(`Use of Ember Data 2+ with Ember 1.13 is deprecated. Please upgrade your version of Ember to 2.0 or higher.`, false, {
+    id: 'ds.version.ember-1-13',
+    until: '3.0.0'
```

Comment:
I think this means we are telling people that we are supporting 1.13 ember until ember data 3.0. I don't know how achievable this goal really is.